Sources/Commands/SwiftTool.swift

```diff
@@ -149,6 +149,10 @@ public class SwiftTool<Options: ToolOptions> {
             option: parser.add(option: "--experimental-use-v4-manifest", kind: Bool.self),
             to: { _, on in Versioning.simulateVersionFour = on })
 
+        binder.bind(
+            option: parser.add(option: "--destination", kind: PathArgument.self),
```

Comment:
I thought we'd add it as an experimental feature first but happy to add text. @aciidb0mb3r / @ddunbar any opinions?